doc/02-installation.md

```diff
@@ -125,6 +125,16 @@ wget https://packages.icinga.com/subscription/rhel/ICINGA-release.repo -O /etc/y
 If you are using RHEL you need to additionally enable the `codeready-builder`
 repository before installing the [EPEL rpm package](https://fedoraproject.org/wiki/EPEL#How_can_I_use_these_extra_packages.3F).
 
+#### RHEL 9
+
+```bash
+ARCH=$( /bin/arch )
+
+subscription-manager repos --enable "codeready-builder-for-rhel-9-${ARCH}-rpms"
+
+dnf install https://dl.fedoraproject.org/pub/epel/epel-release-latest-9.noarch.rpm
+```
+
```

Comment:
... so why not here as well?

Reply:
Would also make it future-proof for RHEL 10 (at least if nothing changes with that version, we'll see in a few years).

Reply:
The new commit improves on that; now everything is RHEL 8 or Later. 

Though I wouldn't count on it being future-proof, but let's see what surprises RHEL 10 will have in store for us.